src/rebar3.erl

```diff
@@ -286,9 +286,14 @@ log_level() ->
             DefaultLevel = rebar_log:default_level(),
             case os:getenv("DEBUG") of
                 D when D == false; D == "" ->
-                    DefaultLevel;
+                    case os:getenv("DIAGNOSTIC") of
+                        Di when Di == false; Di == "" ->
+                            DefaultLevel;
+                        _ ->
+                            DefaultLevel + 2
+                    end;
                 _ ->
```

Comment:
If both are set shouldn't the higher level take precedence, meaning checking for `DIAGNOSTIC` in here too and use it if its set?

Reply:
I imagined that it was Default, DEBUG = default+1, and DIAGNOSTIC = default + 2, i.e. the highest one includes the rest?

Reply:
maybe I'm reading the code wrong, I thought it said that if `DEBUG` is set that it will only be +1, not matter if `DIAGNOSTIC` is set.

Reply:
Oh yeah I see what you mean. I'll fix that so DIAGNOSTIC > DEBUG then.